Cargo.lock

```diff
@@ -3638,6 +3647,7 @@ dependencies = [
 name = "xtask_lintdoc"
 version = "0.0.0"
 dependencies = [
+ "convert_case",
```

Comment:
Do you think we could use our [in-house implementation](https://github.com/biomejs/biome/blob/main/crates/rome_js_analyze/src/utils/case.rs)?

Reply:
I didn't want to bother, the crate is used for a tooling internal script